docs/Manifest Parameters.md

```diff
@@ -905,3 +907,280 @@ The query command can be used to gather information about the system and the Vis
     </config>
 </manifest>
 ```
+
+## Application
+
+#### Source application add Manifest Parameters
+| Tag                                      | Example                                  | Required/Optional | Notes           |
+|:-----------------------------------------|:-----------------------------------------|:-----------------:|:----------------|
+| `<?xml version='1.0' encoding='utf-8'?>` | `<?xml version='1.0' encoding='utf-8'?>` |         R         |                 |
+| `<manifest>`                             | `<manifest>`                             |         R         |                 |
+| `<type></type>`                          | `<type>source</type>`                    |         R         |                 |
+| `<applicationSource>`                    | `<applicationSource>`                    |         R         |                 |
+| `<add>`                                  | `<add>`                                  |         R         |                 |
+| `<gpg>`                                  | `<gpg>`                                  |         R         |                 |
+| `<uri></uri>`                            | `<uri>`                                  |         R         |                 |
+| `<keyname></keyname>`                    | `<keyname></keyname>`                    |         R         |                 | 
+| `</gpg>`                                 | `<gpg>`                                  |         R         |                 |
+| `<repo>`                                 | `</repo>`                                |         R         |                 |
+| `<repos>`                                | `</repos>`                               |         R         |                 |
+| `<source_pkg>`                           | `</source_pkg>`                          |         R         |                 |
+| `</repos>`                               | `</repos>`                               |         R         |                 |
+| `<filename></filename>`                  | `<filename></filename>`                  |         R         |                 |
+| `</repo>`                                | `</repo>`                                |         R         |                 |
+| `</add>`                                 | `</add>`                                 |         R         |                 |
+| `</applicationSource>`                   | `</applicationSource>`                   |         R         |                 |
+| `</manifest>`                            | `</manifest>`                            |         R         |                 |
+
+
+
+
+#### Source application add Manifest Example
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+    <type>source</type>
+    <applicationSource>
+        <add>
+            <gpg>
+                <uri></uri>
```

Comment:
Update the manifest examples in the doc with dummy information (URI/keyname/path etc as applicable)

Reply:
@tsirlapu - were you going to update with dummy information?  I know we talked about it this morning, but I can't remember if you said you were or not.  I can do it, if not.

Reply:
@nmgaston I thought I was not going to update this doc and I need to only work on updating INBC Readme.

Reply:
my impression from the conversation we had was that @nmgaston took the INBC readme task (which is done here) and @tsirlapu took the manifest task (which is not yet done)

Reply:
It should be updated with the manifest now.  I pushed a commit a bit ago.  Let me know if I missed something, but I think it's all there.